minecraft_servers/onthepixel/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "server_name": "onthepixel",
+  "nice_name": "OnThePixel Network",
+  "direct_ip": "play.onthepixel.net",
+  "server_wildcards": [
+    "%.onthepixel.net"
+  ],
+  "social": {
+    "web": "https://onthepixel.net",
```

Comment:
Not available.

Reply:
Hi! Thanks for your review! Is it a problem if I publish the website in one or two days when I’ve finished it?